content/departments/support/tools/support-logging-guide.md

```diff
@@ -0,0 +1,189 @@
+# Application Engineers guide to getting started with Sourcegraph logging, services, and observability
+
+**Note to reader:** This guide assumes little to no prior knowledge of how to troubleshoot distributed systems, gather logs, levels of user access, etc. It is an overview with the goal to provide a starting point for _where to look, why to look there, what you are looking for, and who can provide it._
+
+### At the end of this guide you will...
+
+- Have gained a conceptual understanding of Sourcegraph architecture, services, and the logs they generate
+- Learn how to grep through logs and find what you are looking for
```

Comment:
maybe put grep in the ``grep`` backticks, and maybe say something like:

"Learn how to `grep`/search/parse through logs"

To give them an idea of what it means?

Reply:
Done!